Makefile

```diff
@@ -253,5 +253,9 @@ release-notes:
 	echo "## Issues" >> release_notes.md
 	curl "${GITHUB_API_URL}/${GITHUB_REPOSITORY}/issues?milestone=${MILESTONE_NUM}&state=all" | jq '.[].title' | while read i; do echo "[$i]($MILESTONE_URL)" >> release_notes.md; done
 
-simple-election:
-	poetry run eg e2e --guardian-count=2 --quorum=2 --manifest=data/election_manifest_simple.json --ballots=data/plaintext_ballots_simple.json --spoil-id=25a7111b-4334-425a-87c1-f7a49f42b3a2 --output-file="./election_record.zip"
+eg-e2e-simple-election:
+# to save keys to import append --output-keys="./guardian_private_data.json"
```

Comment:
Is this an todo?

Reply:
Not a todo.  The objective was to add a helpful comment to anyone that might use it, since `output-keys` is a required input field for the `import-ballots` command, and you can only get one of those by outputting it in the e2e command like this.  If you definitely want to delete the sample data and the `eg-e2e-simple-election` command on line 256 then I'll remove this comment too, and add it to the description of https://github.com/microsoft/electionguard-python/issues/574.